packages/trader/src/App/Components/Layout/Header/toggle-cashier.jsx

```diff
@@ -16,31 +16,41 @@ const tabs = {
 const Deposit    = () => import('App/Containers/CashierModal/deposit.jsx');
 const Withdrawal = () => import('App/Containers/CashierModal/withdrawal.jsx');
 
-const modal_content = [
-    {
-        icon : 'IconDepositSmall',
-        label: localize('Deposit'),
-        // eslint-disable-next-line react/display-name
-        value: () => (
-            <Lazy
-                ctor={Deposit}
-                should_load={true}
-                has_progress={true}
-            />
-        ),
-    }, {
-        icon : 'IconWithdrawalSmall',
-        label: localize('Withdrawal'),
-        // eslint-disable-next-line react/display-name
-        value: () => (
-            <Lazy
-                ctor={Withdrawal}
-                should_load={true}
-                has_progress={true}
-            />
-        ),
-    },
-];
+const ModalContent = () => {
+    const content = [
+        {
+            icon : 'IconDepositSmall',
+            label: localize('Deposit'),
+            // eslint-disable-next-line react/display-name
+            value: () => (
+                <Lazy
+                    ctor={Deposit}
```

Comment:
Would you rename `Deposit` to `loadDepositComponent`?

Reply:
@reza-fs its not mine. negar is working on it. i think you can add this on her PR.